src/mfa/interfaces/totp.interface.ts

```diff
@@ -1,28 +1,15 @@
-interface TotpObject {
+export interface Totp {
   issuer: string;
   user: string;
-  qrCode: string;
-  secret: string;
-  uri: string;
+  qrCode?: string;
```

Comment:
are these optional or required for standalone MFA responses?

Reply:
for both standalone and userland mfa, the secret properties (qr code, secret, and uri) are used in some cases and not used in others. 

so they should be optional for both

Reply:
ah ok, so seems like the standalone MFA code for retrieving a totp factor was broken? 🤔  (asking for context so i know what to put in the changelog)

Reply:
yes! the standalone mfa api was inacurrately saying that there was always access to these responses even in methods that didn't have them